src/Microsoft.Health.Fhir.Core/Features/Search/Expressions/ExpressionRewriter.cs

```diff
@@ -38,7 +38,16 @@ public virtual Expression VisitChained(ChainedExpression expression, TContext co
                 return expression;
             }
 
-            return new ChainedExpression(resourceTypes: expression.ResourceTypes, referenceSearchParameter: expression.ReferenceSearchParameter, targetResourceTypes: expression.TargetResourceTypes, reversed: expression.Reversed, expression: visitedExpression);
+            /*SortRewriter will return null expression if it finds _sort param same as on of filter params.
+             But in case of chained names we do need to keep it. #94770*/
```

Comment:
I would suggest removing the Bug ID #94770 from this comment. 
This Bug ID is only usable in our internal repository :)
From an open-source perspective, this ID is not required :)

Reply:
done